Description: in Debian, the package is called "maildir-utils", not "mu",
 and so the doc's are installed into ${datadir}/doc/maildir-utils.
Author: Norbert Preining <preining@debian.org>
Author: Jeremy Sowden <jeremy@azazel.net>
Last-Update: 2023-11-03
Forwarded: not-needed

--- a/mu4e/meson.build
+++ b/mu4e/meson.build
@@ -26,7 +26,7 @@
   configuration: {
       'VERSION'           : meson.project_version(),
       'MU_VERSION_EXTRA'  : version_extra,
-      'MU_DOC_DIR'        : join_paths(datadir, 'doc', 'mu'),
+      'MU_DOC_DIR'        : join_paths(datadir, 'doc', 'maildir-utils'),
     })
 
 mu4e_pkg_desc = configure_file(
@@ -134,7 +134,7 @@
 install_data(mu4e_srcs, install_dir: mu4e_lispdir)
 
 # install mu4e-about.org
-install_data('mu4e-about.org', install_dir : join_paths(datadir, 'doc', 'mu'))
+install_data('mu4e-about.org', install_dir : join_paths(datadir, 'doc', 'maildir-utils'))
 
 if makeinfo.found()
   custom_target('mu4e_info',
--- a/meson.build
+++ b/meson.build
@@ -283,7 +283,7 @@
 ################################################################################
 # install some data files
 install_data('NEWS.org', 'IDEAS.org',
-             install_dir : join_paths(datadir, 'doc', 'mu'))
+             install_dir : join_paths(datadir, 'doc', 'maildir-utils'))
 
 ################################################################################
 # subdirs
